glide.yaml

```diff
@@ -101,3 +101,388 @@ import:
   version: c9de0b92586aaba092c2a231cb7384354b178544
 - package: github.com/lestrrat/go-jspointer
   version: f4881e611bdbe9fb413a7780721ef8400a1f2341
+
```

Comment:
can remove the `# pinned` ones above... that was stopgap to get a minimal diff on the rebase

Reply:
> can remove the # pinned ones above and regenerate... those were stopgap to get a minimal diff on the rebase

done